htdocs/config2/settings-sample-dev.inc.php

```diff
@@ -25,3 +25,8 @@
 // database names
 $opt['db']['placeholder']['db'] = 'opencaching';
 $opt['db']['placeholder']['tmpdb'] = 'octmp';
+
+/**
+ * for more config examples (blog, forum) please have a look in commit 7a7e68a8af9a331da6002dca52fe3bff052c661a
+ *
+ */
```

Comment:
Disagree. These are prepared settings which belong to the current OC code distribution.